packages/runtime-common/realm.ts

```diff
@@ -824,6 +836,15 @@ export class Realm {
     }
     let { doc: card } = maybeError;
     card.data.links = { self: url.href };
+
+    let foundPath = this.paths.local(url);
+    if (localPath !== foundPath) {
```

Comment:
sorry ignore that previous comment. i see that we are trying to validate that if the .json extension is requested that we do due dillegence to see if it's valid before sending a redirect.